### DISCUSSION TITLE
✨ scrutiny

### DISCUSSION THREAD

**Discussion by Artzert2215** [runtipi/runtipi-appstore#4573] [Requests]
### Description

WebUI for smartd S.M.A.R.T monitoring

### Documentation link

https://github.com/AnalogJ/scrutiny/tree/master/docs

### Last application release & date

v0.8.1 (8/4/2024)

### Application license

MIT License

### Source code repository link

https://github.com/AnalogJ/scrutiny

### Docker image link

https://github.com/AnalogJ/scrutiny/pkgs/container/scrutiny

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by OriginalSimon** [comment]
A very much needed app.
